bridge/client/app/_models/sequence.ts

```diff
@@ -275,7 +278,9 @@ export class Sequence implements ISequence {
 
   public getIcon(stageName?: string): DtIconType {
     const stage = stageName ? this.getStage(stageName) : this.stages[this.stages.length - 1];
-    return stage?.latestEvent?.type
+    return this.isPaused()
```

Comment:
Please also extract this statement into a function, so it can easily be tested and maintained.